pkgdb2client/utils.py

```diff
@@ -187,6 +187,18 @@ def __get_fas_user_by_email(email_address):
     return user
 
 
+def get_fasinfo(email):
+    ''' Get fas username and build a name string for the user like:
+        Human Name (fas_username) <email>
+    '''
+
+    fas_user = __get_fas_user_by_email(email)
+    fas_username = fas_user.get("username", "").encode("utf-8")
+    human_name = fas_user.get("human_name", "").encode("utf-8")
+    full_info = "{0} ({1}) <{2}>".format(human_name, fas_username, email)
```

Comment:
There are cases where an email in bugzilla is not linked to a FAS account, these people have a dedicated mapping in python-fedora that's overridden by a hotfix on pkgdb0{1,2} [1]. So I wonder if we shouldn't check what ``fas_user`` is and if it is ``None``, then return a sensible string using the email.

[1] https://infrastructure.fedoraproject.org/infra/ansible/roles/pkgdb2/files/fas2.py